src/components-v2/layouts/ListContentLayout/List.jsx

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+import React from 'react'
+import styled from 'styled-components'
+import cn from 'classnames'
+import { withStyles } from '@material-ui/core/styles'
+
+// TODO: move these styles to a generic place
+import styles from './style'
+
+const Wrapper = styled.div``
+
+const Item = styled.div`
+  border-bottom: solid 2px rgb(232, 231, 230);
+
+  .container {
+    display: flex;
+    align-items: flex-end;
+  }
+`
+
+const IconImg = styled.img`
+  width: 20px;
+  margin-right: 10px;
+`
+type Props = {
+  items: Array<{
+    id: string,
+    name: string,
+    iconUrl?: string,
+  }>,
+  activeItem: string,
+  onItemClick: () => void,
+  classes: Object,
+}
+
+const List = ({ items, activeItem, onItemClick, classes }: Props) => {
+  return (
+    <Wrapper>
+      {items.map(i => (
+        <Item
+          key={i.id}
+          className={cn(
+            classes.menuOption,
+            activeItem === i.id && classes.active
```

Comment:
i thought with styled components we get rid of classes, don't we?

Reply:
I'm using clases becase there are some styles that are already defined in the theme. it's to avoid replicate styles. In a later refactor, we can migrate all that to styled-components